code/src/index.css

```diff
@@ -1,13 +1,108 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+  font-family: 'Trispace', sans-serif;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+.thought-card, .thoughts-form {
+  margin: 20px auto;
+  border: 2px solid black;
+  width: 500px;
+  box-shadow: 5px 5px;
+  padding: 20px;
 }
+
+.thought-card {
+  word-wrap: break-word;
+}
+
+.thoughts-form {
+  background-color: #f2f0f0;
+  display: flex;
+  flex-direction: column;
+}
+
+.form-title {
+  margin-top: 0;
+  font-weight: bold;
+}
+
+.form-button {
+  margin-top: 10px;
+  width: 250px;
+  background-color: #ffadad;
+  border: none;
+  padding: 10px;
+  border-radius: 30px;
+  font-family: 'Trispace', sans-serif;
+  font-weight: bold;
+  cursor: pointer;
+}
+
+textarea {
+  font-size: 17px;
+  font-family: 'Trispace', sans-serif;
+  width: 470px;
+  height: 80px;
+  max-width: 470px;
+  max-height: 150px;
+}
+
+.text-counter {
+  font-size: 12px;
+}
+
+.heart-button {
+  padding: 12px;
+  margin-right: 7px;
+  border: none;
+  border-radius: 50%;
+}
+
+.time-created {
+  color: #a09993;
+}
+
+.heart-time-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  width: 500px;
+}
+
+.posts-liked-counter {
+  margin: 20px auto;
+  border: 2px solid black;
+  width: 500px;
+  box-shadow: 5px 5px;
+  padding: 20px;
+  background-color: #ffadad;
+  font-weight: bold;
+}
+
+/*Media Queries*/
+/*Mobile*/
+@media (max-width: 666px) {
```

Comment:
Not very common breakpoint, if you thought about mobile break, I would suggest `768px`